libs/wingsdk/src/shared-aws/api.ts

```diff
@@ -68,3 +70,28 @@ export class Api {
     );
   }
 }
+
+/**
+ * A helper class for working with AWS api endpoint handlers.
+ */
+export class ApiEndpointHandler {
+  /**
+   * Returns a `cloud.Function` handler for handling requests from a `cloud.Api`.
```

Comment:
Technically this is specific to aws, the handler returned by this is not a generic cloud.Function handler.